src/Sylius/Bundle/CoreBundle/Fixture/ShippingMethodFixture.php

```diff
@@ -37,7 +37,7 @@ protected function configureResourceNode(ArrayNodeDefinition $resourceNode): voi
                 ->scalarNode('description')->cannotBeEmpty()->end()
                 ->scalarNode('zone')->cannotBeEmpty()->end()
                 ->booleanNode('enabled')->end()
-                ->scalarNode('category')->end()
+                ->scalarNode('shipping_category')->end()
```

Comment:
I think we should fix the example factory to allow for `category` instead (as shipping method has `taxCategory` and `category` relations, there's no `shippingCategory`).

Reply:
This change would make any fixture with `category` defined failing.